snapshot/cookie/.pre-commit-config.yaml

```
@@ -2,7 +2,7 @@
 default_stages: [push]
 repos:
   - repo: https://github.com/crate-ci/typos
-    rev: v1.19.0
+    rev: typos-v0.10.26
```

```suggestion
    rev: v1.23.2
```    

see https://github.com/crate-ci/typos/issues/390